Sources/System/FileSystem/Stat.swift

```diff
@@ -0,0 +1,749 @@
+  @_alwaysEmitIntoClient
+  public var blocksAllocated: Int64 {
+    get { Int64(rawValue.st_blocks) }
+    set { rawValue.st_blocks = numericCast(newValue) }
+  }
+
+  /// Total size allocated, in bytes
+  ///
+  /// The semantics of this property are tied to the underlying C `st_blocks` field,
+  /// which can have file system-dependent behavior.
+  ///
+  /// - Note: Calculated as `512 * blocksAllocated`.
+  @_alwaysEmitIntoClient
+  public var sizeAllocated: Int64 {
+    512 * blocksAllocated
+  }
+
+  // NOTE: "st_" property names are used for the `timespec` properties so
+  // we can reserve `accessTime`, `modificationTime`, etc. for potential
+  // `UTCClock.Instant` properties in the future.
+
+  /// Time of last access, given as a C `timespec` since the Epoch.
+  ///
+  /// The corresponding C property is `st_atim` (or `st_atimespec` on Darwin).
+  @_alwaysEmitIntoClient
+  public var st_atim: timespec {
+    get {
+      #if SYSTEM_PACKAGE_DARWIN
+      rawValue.st_atimespec
+      #else
+      rawValue.st_atim
+      #endif
+    }
+    set {
+      #if SYSTEM_PACKAGE_DARWIN
+      rawValue.st_atimespec = newValue
+      #else
+      rawValue.st_atim = newValue
+      #endif
+    }
+  }
+
+  /// Time of last modification, given as a C `timespec` since the Epoch.
+  ///
+  /// The corresponding C property is `st_mtim` (or `st_mtimespec` on Darwin).
+  @_alwaysEmitIntoClient
+  public var st_mtim: timespec {
+    get {
+      #if SYSTEM_PACKAGE_DARWIN
+      rawValue.st_mtimespec
+      #else
+      rawValue.st_mtim
+      #endif
+    }
+    set {
+      #if SYSTEM_PACKAGE_DARWIN
+      rawValue.st_mtimespec = newValue
+      #else
+      rawValue.st_mtim = newValue
+      #endif
+    }
+  }
+
+  /// Time of last status (inode) change, given as a C `timespec` since the Epoch.
+  ///
+  /// The corresponding C property is `st_ctim` (or `st_ctimespec` on Darwin).
+  @_alwaysEmitIntoClient
+  public var st_ctim: timespec {
+    get {
+      #if SYSTEM_PACKAGE_DARWIN
+      rawValue.st_ctimespec
+      #else
+      rawValue.st_ctim
+      #endif
+    }
+    set {
+      #if SYSTEM_PACKAGE_DARWIN
+      rawValue.st_ctimespec = newValue
+      #else
+      rawValue.st_ctim = newValue
+      #endif
+    }
+  }
+
+  #if SYSTEM_PACKAGE_DARWIN || os(FreeBSD)
+  /// Time of file creation, given as a C `timespec` since the Epoch.
+  ///
+  /// The corresponding C property is `st_birthtim` (or `st_birthtimespec` on Darwin).
+  /// - Note: Only available on Darwin and FreeBSD.
+  @_alwaysEmitIntoClient
+  public var st_birthtim: timespec {
+    get {
+      #if SYSTEM_PACKAGE_DARWIN
+      rawValue.st_birthtimespec
+      #else
+      rawValue.st_birthtim
+      #endif
+    }
+    set {
+      #if SYSTEM_PACKAGE_DARWIN
+      rawValue.st_birthtimespec = newValue
+      #else
+      rawValue.st_birthtim = newValue
+      #endif
+    }
+  }
+  #endif
+
+  // TODO: Investigate changing time properties to UTCClock.Instant once available.
```

Comment:
Let's remove these in a standalone commit so that they're easy to find later, instead of leaving commented-out code.

Reply:
Done! https://github.com/apple/swift-system/pull/256/commits/501bb7d052f5b41a6bbb0dad1db611f5b527cafc